ocaml/tests/cancel_tests.ml

```diff
@@ -1,431 +0,0 @@
-(*
- * Copyright (C) Citrix Systems Inc.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published
- * by the Free Software Foundation; version 2.1 only. with the special
- * exception on linking described in file LICENSE.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Lesser General Public License for more details.
- *)
-open Client
-open Pervasiveext
-
-let debug (fmt: ('a , unit, string, unit) format4) =
-  (* Convert calendar time, x, to tm in UTC *)
-  let of_float x =
-    let time = Unix.gmtime x in
-    Printf.sprintf "%04d%02d%02dT%02d:%02d:%02dZ"
-      (time.Unix.tm_year+1900)
-      (time.Unix.tm_mon+1)
-      time.Unix.tm_mday
-      time.Unix.tm_hour
-      time.Unix.tm_min
-      time.Unix.tm_sec in
-
-  Printf.kprintf (fun s -> Printf.printf "%s [%d] %s\n" (of_float (Unix.gettimeofday ())) (Thread.id (Thread.self ())) s; flush stdout) fmt
-let host = ref "127.0.0.1"
-let port = ref 80
-let username = ref "root"
-let password = ref ""
-let vm = ref ""
-
-let make_rpc ?dbg () xml =
-  let open Xmlrpc_client in
-  let http = {
-    (xmlrpc ~version:"1.0" "/") with
-    Http.Request.additional_headers = Opt.default [] (Opt.map (fun dbg -> [ "X-Http-other-config-dbg", dbg ]) dbg)
-  } in
-  XMLRPC_protocol.rpc ~srcstr:"graph" ~dststr:"xapi" ~transport:(TCP(!host, !port)) ~http xml
-
-let wait_for_guest_agent ~rpc ~session_id ~vm =
```

Comment:
This looks like a useful test, should we add it to quicktest instead of deleting it (assuming it stil works)?

Reply:
We can create a ticket for that.